Makefile

```diff
@@ -121,11 +125,16 @@ docker-buildx: ## Build and push docker image for the manager for cross-platform
 	rm Dockerfile.cross
 
 .PHONY: build-installer
-build-installer: manifests generate kustomize ## Generate a consolidated YAML with CRDs and deployment.
+build-installer: manifests generate kustomize build-csi-rbac ## Generate a consolidated YAML with CRDs and deployment.
```

Comment:
You don't need this to do this.
 You can, instead, add the new csi folder into `default/kusomizatoin.yaml` as all other folders.